cmake/ConfigureCompiler.cmake

```diff
@@ -228,7 +234,20 @@ else()
   endif()
   if (GCC)
     add_compile_options(-Wno-pragmas)
-
+    set(USE_AVX ON CACHE BOOL "Enable AVX instructions")
```

Comment:
having this twice is a bit ugly. Cached variables should go to the top of the file (so they're easier to spot - and they are not scoped anyways).

Reply:
fixed.  thanks.